reactome_analysis_api/reactome_analysis_api/controllers/datasets_controller.py

```diff
@@ -201,21 +207,26 @@ def load_data(resourceId, parameters):  # noqa: E501
 
 def get_search_species():  # noqa: E501
     """Returns the available species presented in the available datasets.
-
      # noqa: E501
-
     :rtype: list
     """
-    raise NotImplementedError()
+    try:
+        searcher = PublicDatasetSearcher(SEARCH_INDEX_PATH)
```

Comment:
Hier gäbe auch auch die Möglichkeit, den Searcher nur 1x beim Start der Applikation zu laden (was wahrscheinlich ein wenig schneller wäre).

Dazu kann man zusätzliche member variablen der Flask applikation hinzufügen. Können wir gerne noch besprechen.